GRADLE_PLUGIN.md

```diff
@@ -1,10 +1,268 @@
 # Kotlin/Native Gradle plugin
 
-### IMPORTANT NOTICE
+Since 1.3.40, a separate Gradle plugin for Kotlin/Native is deprecated in favor of the `kotlin-multiplatform` plugin.
+This plugin provides an IDE support along with support of the new multiplatform project model introduced in Kotlin 1.3.0.
+Below you can find a short list of differences between `kotlin-platform-native` and `kotlin-muliplatform` plugins. 
+For more information see the `kotlin-muliplatform` [documentation page](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html).
+For `kotlin-platform-native` reference see the [corresponding secion](#kotlin-platform-native-reference).
 
-This document describes Kotlin/Native experimental Gradle plugin, which is not the plugin yet supported by IDE
-or in multiplatform projects. See MPP Gradle plugin [documentation](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html)
-for more information.
+### Applying the multiplatform plugin
+
+To apply the `kotlin-multiplatform` plugin, just add the following snippet into your build script:
+
+<div class="sample" markdown="1" theme="idea" mode="groovy">
+
+```groovy
+plugins {
+    id 'org.jetbrains.kotlin.multiplatform' version '1.3.40'
```

Comment:
If kotlinlang.org is the only place where this documentation is presented, then you may use `{{ site.data.releases.latest.version }}` Instead of a particular Kotlin version.

Reply:
Also consider using the syntax that is valid in both Groovy and Kotlin where it is feasible. This ensures that copying a sample into the build script works in more cases. Here, it would be `id("org.jetbrains.kotlin.multiplatform") version "..."`

Reply:
> If kotlinlang.org is the only place where this documentation is presented, then you may use {{ site.data.releases.latest.version }} Instead of a particular Kotlin version.

Sometimes we use GitHub links to this documentation, so I think it's better to leave explicit version here.

> Also consider using the syntax that is valid in both Groovy and Kotlin where it is feasible. 

Fixed.